server/app/db/crud/bots.py

```diff
@@ -0,0 +1,331 @@
+"""Database layer for bots"""
+
+import logging
+from typing import Any, List, Tuple, Union
+
+import db.schemas.bots as bots_schema
+from aiomysql.cursors import Cursor
+from pydantic import ValidationError
+from pypika import Criterion, MySQLQuery, Table
+
+from .base import log_query, log_query_with_arguments
+from .bot_types import get_bot_type_with_given_name
+
+bots, bot_types = Table("bots", "bot_types")
+
+
+async def create_bot(
```

Comment:
adding it here for later reference, so we should register bot user in dalal server while inserting in db right ?

Reply:
do we need to add it here or should we add it some place else like ```api/controllers```?

Reply:
can we create something like this to handle all grpc unary calls

Reply:
yea we can do that in grpc manager